<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComplaintDialog</class>
 <widget class="QDialog" name="ComplaintDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>175</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>175</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: rgb(245, 245, 245);
	border-radius: 20px;
	height: 200px;
	width: 300px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="descriptionLabel">
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(65, 47, 50);
	padding-left: 10px;
	height: 30px;
}</string>
     </property>
     <property name="text">
      <string>Опишите жалобу в поле ниже.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(245, 245, 245);
	color: rgb(60, 45, 50);
	border: 5px solid rgb(60, 45, 50);
	border-radius: 20;
}

QPushButton:hover {
	background-color: rgb(60, 45, 50);
	color: rgb(245, 245, 245);
}</string>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="sendButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(245, 245, 245);
	color: rgb(60, 45, 50);
	border: 5px solid rgb(60, 45, 50);
	border-radius: 20;
}

QPushButton:hover {
	background-color: rgb(60, 45, 50);
	color: rgb(245, 245, 245);
}</string>
     </property>
     <property name="text">
      <string>Отправить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="complaintEdit">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background-color: rgb(245, 245, 245);
	height: 50px;
	color: rgb(60, 47, 50);
	margin-top: 5px;
	padding-left: 10px;
	border-bottom: 3px solid rgb(60, 47, 55);
	border-radius: 1px;
	font-size: 20px;
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
